articles/quickstart/native/ios-objc/index.yml

```diff
@@ -1,28 +1,21 @@
 title: iOS Objective-C
-hybrid: false
 language:
   - Objective-C
 image: /media/platforms/ios.png
 tags:
   - quickstart
-snippets:
-  dependencies: native-platforms/ios-objc/dependencies
-  setup: native-platforms/ios-objc/setup
-  use: native-platforms/ios-objc/use
 alias:
   - ios
   - objective-c
   - iphone
   - ipad
 seo_alias: ios-objc
+snippets:
+  setup: native-platforms/ios-objc/setup_wrapper
 articles:
-  - 00-introduction
-  - 01-login
-  - 02-custom-login
-  - 03-session-handling
-  - 04-user-profile
```

Comment:
FYI. I see that you have merged the "session handling" (refreshing tokens) and "user profile" (changing user_metadata.country) articles into the one you now call "user-sessions". In my case i have them separate.